### PR TITLE
few-shot示例保留向量查询最佳匹配结果

### PR DESCRIPTION
**Pull request by chsumu**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Additional information

Any additional information, configuration or data that might be necessary to reproduce the issue.

### CODE REVIEW

File: chat/server/src/main/java/com/tencent/supersonic/chat/server/service/impl/MemoryServiceImpl.java [hunk 1]
```diff
@@ -91,8 +92,10 @@ public List<ChatMemoryDO> getMemories(ChatMemoryFilter chatMemoryFilter) {
     @Override
     public List<ChatMemoryDO> getMemoriesForLlmReview() {
         QueryWrapper<ChatMemoryDO> queryWrapper = new QueryWrapper<>();
-        queryWrapper.lambda().eq(ChatMemoryDO::getStatus, MemoryStatus.PENDING)
-                .isNull(ChatMemoryDO::getLlmReviewRet);
+        queryWrapper.lambda()
+                .eq(ChatMemoryDO::getStatus, MemoryStatus.PENDING)
+                .eq(ChatMemoryDO::getLlmReviewRet, MemoryReviewResult.POSITIVE)
+                .eq(ChatMemoryDO::getHumanReviewRet, MemoryReviewResult.POSITIVE);
         return chatMemoryRepository.getMemories(queryWrapper);
```
Comment (jerryjzhang) [inline chat/server/src/main/java/com/tencent/supersonic/chat/server/service/impl/MemoryServiceImpl.java]: 这里需要获取所有pending的memory交给LLM做review，如果加这个状态过滤逻辑就不对了

**Review by jerryjzhang — COMMENTED** [review]
增加score是好的想法，只不过当前如果无法从memory里召回到few shot样例，就会采用系统默认的样例，相比较起来虽然memory样例有可能语义不太相关，但总归是属于同一个domain，会比系统默认样例要更合适一些

File: chat/server/src/main/java/com/tencent/supersonic/chat/server/service/impl/MemoryServiceImpl.java (hunk 1 above)
Comment (chsumu) [reply chat/server/src/main/java/com/tencent/supersonic/chat/server/service/impl/MemoryServiceImpl.java]: 这里按原有的逻辑实测会污染向量库，LLM 会判断错误，还是建议增加管理员判断逻辑

Comment (chsumu) [reply chat/server/src/main/java/com/tencent/supersonic/chat/server/service/impl/MemoryServiceImpl.java]: 抱歉，我的错误，应该在 MemoryReviewTask 里面做过滤逻辑的

**Pull request closed without merging**